src/model/test/Space_GTest.cpp

```diff
@@ -2720,52 +2543,224 @@ TEST_F(ModelFixture, ExposedPerimeter) {
+      EXPECT_NEAR(exposedPerimeter, 26, 0.01);
     } else if (spacename == "Mechanical 114") {
-      ASSERT_EQ(nOverlaps, 0);
-      ASSERT_NEAR(exposedPerimeter, 0, 0.01);
+      EXPECT_EQ(nOverlaps, 0);
+      EXPECT_NEAR(exposedPerimeter, 0, 0.01);
     } else if (spacename == "Media Center 116") {
-      ASSERT_EQ(nOverlaps, 3);
-      ASSERT_NEAR(exposedPerimeter, 198, 0.01);
+      EXPECT_EQ(nOverlaps, 3);
+      EXPECT_NEAR(exposedPerimeter, 198, 0.01);
     } else if (spacename == "Offices 117") {
-      ASSERT_EQ(nOverlaps, 2);
-      ASSERT_NEAR(exposedPerimeter, 138, 0.01);
+      EXPECT_EQ(nOverlaps, 2);
+      EXPECT_NEAR(exposedPerimeter, 138, 0.01);
     } else if (spacename == "Corridor 104" || spacename == "Corridor 110") {
-      ASSERT_EQ(nOverlaps, 1);
-      ASSERT_NEAR(exposedPerimeter, 10, 0.01);
+      EXPECT_EQ(nOverlaps, 1);
+      EXPECT_NEAR(exposedPerimeter, 10, 0.01);
     } else if (spacename == "Corridor 107") {
-      ASSERT_EQ(nOverlaps, 1);
-      ASSERT_NEAR(exposedPerimeter, 28, 0.01);
+      EXPECT_EQ(nOverlaps, 1);
+      EXPECT_NEAR(exposedPerimeter, 28, 0.01);
     } else if (spacename == "Cafeteria 120") {
-      ASSERT_EQ(nOverlaps, 2);
-      ASSERT_NEAR(exposedPerimeter, 118, 0.01);
+      EXPECT_EQ(nOverlaps, 2);
+      EXPECT_NEAR(exposedPerimeter, 118, 0.01);
     } else {
       // Makes sure we've checked all 20 spaces and got the names right
       ASSERT_FALSE(true);
     }
   }
 }
 
-#endif
+/*****************************************************************************************************************************************************
+*                                                           D I S A B L E D    T E S T S                                                            *
+*****************************************************************************************************************************************************/
+
+// This takes 20secs but passes: TODO enable? (this is a bit too long)
+TEST_F(ModelFixture, DISABLED_Issue_1322) {
+
+  osversion::VersionTranslator translator;
+  openstudio::path modelPath = resourcesPath() / toPath("model/7-7_Windows_Complete.osm");
+  model::OptionalModel model = translator.loadModel(modelPath);
+  EXPECT_TRUE(model);
+
+  SpaceVector spaces = model->getModelObjects<Space>();
+
+  intersectSurfaces(spaces);
+  matchSurfaces(spaces);
+}
+
+// This takes 5secs but passes: TODO: enable? (borderline too long to pass)
+TEST_F(ModelFixture, DISABLED_Issue_1683) {
+
+  osversion::VersionTranslator translator;
+  openstudio::path modelPath = resourcesPath() / toPath("model/15023_Model12.osm");
+  model::OptionalModel model = translator.loadModel(modelPath);
+  EXPECT_TRUE(model);
+
+  SpaceVector spaces = model->getConcreteModelObjects<Space>();
+  intersectSurfaces(spaces);
+  matchSurfaces(spaces);
+
+  // openstudio::path outPath = resourcesPath() / toPath("model/15023_Model12_after.osm");
+  // model->save(outPath, true);
+}
```

Comment:
I'm not sure what to do with these two. They pass, but they are slow. I know you reported one took 16minutes originally, but now it still takes 20secs which is too long for my taste.

Is there anything in these two files that **isn't** already covered in a model GTest? If so, can we do that? Then we can just remove the two OSM and tests here

@ggartside @tijcolem @kbenne thoughts?